packages/menu/src/spectrum-menu.css

```diff
@@ -43,6 +166,167 @@ governing permissions and limitations under the License.
     overflow: auto;
 }
 
+:host([size='s']) {
+    --spectrum-menu-item-min-height: var(--spectrum-component-height-75);
+    --spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-75);
+    --spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-75);
+    --spectrum-menu-item-label-font-size: var(--spectrum-font-size-75);
+    --spectrum-menu-item-label-text-to-visual: var(
+        --spectrum-text-to-visual-75
+    );
+    --spectrum-menu-item-label-inline-edge-to-content: var(
+        --spectrum-component-edge-to-text-75
+    );
+    --spectrum-menu-item-top-edge-to-text: var(
+        --spectrum-component-top-to-text-75
+    );
+    --spectrum-menu-item-bottom-edge-to-text: var(
+        --spectrum-component-bottom-to-text-75
+    );
+    --spectrum-menu-item-text-to-control: var(--spectrum-text-to-control-75);
+    --spectrum-menu-item-description-font-size: var(--spectrum-font-size-50);
+    --spectrum-menu-section-header-font-size: var(--spectrum-font-size-75);
+    --spectrum-menu-section-header-min-width: var(
+        --spectrum-component-height-75
+    );
+    --spectrum-menu-item-selectable-edge-to-text-not-selected: var(
+        --spectrum-menu-item-selectable-edge-to-text-not-selected-small
+    );
+    --spectrum-menu-item-checkmark-height: var(
+        --spectrum-menu-item-checkmark-height-small
+    );
+    --spectrum-menu-item-checkmark-width: var(
+        --spectrum-menu-item-checkmark-width-small
+    );
+    --spectrum-menu-item-top-to-checkmark: var(
+        --spectrum-menu-item-top-to-selected-icon-small
+    );
+    --spectrum-menu-back-icon-margin: var(
+        --spectrum-navigational-indicator-top-to-back-icon-small
+    );
+}
+
+:host,
+:host {
+    --spectrum-menu-item-min-height: var(--spectrum-component-height-100);
+    --spectrum-menu-item-icon-height: var(--spectrum-workflow-icon-size-100);
+    --spectrum-menu-item-icon-width: var(--spectrum-workflow-icon-size-100);
+    --spectrum-menu-item-label-font-size: var(--spectrum-font-size-100);
+    --spectrum-menu-item-label-text-to-visual: var(
+        --spectrum-text-to-visual-100
+    );
```

Comment:
I imagine this is the conversion tool but add new lines inside of var functions can't be safely minified because custom properties can contain anything.